ops/pentest/main.tf

```diff
@@ -102,3 +102,8 @@ module "app_service_autoscale" {
 
   tags = local.management_tags
 }
+
+module "logic_app_workflow" {
+  source                    = "../services/alerts/alert_app"
+  azure_alert_slack_webhook = ""
```

Comment:
@alismx I'm still not sure what to add as the value here in order to pull in the value of the Github secret named `AZURE_ALERT_SLACK_WEBHOOK`

Reply:
What if instead of passing it in like this, you put this secret in the Azure vault and pull it via a terraform data object and use it that way? [Like how we do here.](https://github.com/CDCgov/prime-simplereport/blob/fc71af9f628c3cda0fa2ceeab7316538beb55f80/ops/pentest/_data.tf#L50)

Once you have it, it seems like we might be able to use it [like this.](https://github.com/CDCgov/prime-simplereport/blob/fc71af9f628c3cda0fa2ceeab7316538beb55f80/ops/pentest/api.tf#L32)